packaging/rpm/paack.py

```diff
@@ -89,6 +89,9 @@
 
 %%_POST_%%
 
+# rpm-ostree seems to be unhappy about this, but crio storage (read-only) works without it
+# mknod -m 600 "%{imageStore}/overlay/backingFsBlockDev" b 253 0
```

Comment:
should this be commented out?

Reply:
From side-bar with @ggiguash 
>Yes, that's the error which [@majopela](https://coreos.slack.com/team/UEFFLKJAJ) tried to fix.

Reply:
Yes, it's fine, rpm install didn't work with rpm-ostree if I added it in post.

Reply:
Thanks for reviewing and checking around.